ng2-components/ng2-alfresco-documentlist/src/data/share-datatable-adapter.ts

```diff
@@ -176,8 +176,15 @@ export class ShareDataTableAdapter implements DataTableAdapter {
         this.imageResolver = resolver;
     }
 
-    private sortRows(rows: DataRow[], sorting: DataSorting, options?: Intl.CollatorOptions) {
+    private sortRows(rows: DataRow[], sorting: DataSorting) {
+        const options: Intl.CollatorOptions = {};
```

Comment:
Just a check, does it work ok for FF and Safari 11?

Reply:
Tested on Safari 11.0 (13604.1.38.1.6) and Firefox 55.0.3. Seems to work